service/builder.go

```diff
@@ -22,31 +22,25 @@ package service
 
 import "github.com/pkg/errors"
 
+// WithModules is a helper to create a service without any options
+func WithModules(modules ...ModuleCreateFunc) *Builder {
```

Comment:
@alsamylkin The concern here is that building service needs `modules` and `options`, and people like to to optional arguments to pass them in. This results in split them into 2 functions anyway.

@anuptalwalkar Plan to make this standard building method. Another possibility here is to use a config struct. This is beyond the scope of this diff though.